sdks/java/extensions/sketching/src/main/java/org/apache/beam/sdk/extensions/sketching/ApproximateDistinct.java

```diff
@@ -258,10 +258,32 @@
       abstract GloballyDistinct<InputT> build();
     }
 
+    /**
+     * Returns a new {@link PTransform} with a new precision {@code p}.
```

Comment:
These javadoc changes seem unrelated to the current PR but it's fine to keep them.

However, please modify them according to the PTransform style guide: it's not necessary to say "Returns a new PTransform..." - this is true for all builder methods of all PTransforms in Beam and all its libraries, so it does not need to be repeated. Just say something like "Sets the precision" - throughout.